### PR TITLE
improvement: ZENKO-1920 count items cron schedule

### PR DESCRIPTION
**Pull request by rahulreddy**
improvement: ZENKO-1920 count items cron schedule

### CODE REVIEW

**Review by ssalaues — APPROVED** [review]

**Review by tmacro — APPROVED** [review]

**Review by jonathan-gramain — APPROVED** [review]

File: kubernetes/zenko/values.yaml [hunk 1]
```diff
@@ -72,7 +72,7 @@ reporting:
   enabled: true
   successfulJobsHistory: 1
   countItems:
-    schedule: "* */1 * * *"
+    schedule: "@hourly"
```
Comment (bennettbuchanan) [inline kubernetes/zenko/values.yaml:75]: I like this and didn't know about this cron format. It is much more interpretable!

**Review by bennettbuchanan — APPROVED** [review]

**Review by anurag4DSB — APPROVED** [review]

**Comment by rahulreddy** [comment]
@bert-e approve

**Comment by ssalaues** [comment]
@bert-e bypass_build_status

**Comment by rahulreddy** [comment]
Thanks @ssalaues !

**Pull request merged**